dagshub/auth/tokens.py

```diff
@@ -4,6 +4,7 @@
 import threading
 import traceback
 from typing import Optional, Dict, List, Set, Union
+from multiprocessing import AuthenticationError
```

Comment:
Very weird module to import from, either create a new error class, or use RuntimeError

Examples: https://github.com/DagsHub/client/blob/master/dagshub/data_engine/model/errors.py

Reply:
that was weird to me as well, took it from `dagshub/streaming/filesystem.py`